apps/tangle-dapp/data/NominatorStats/useTotalPayoutRewards.ts

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { BN } from '@polkadot/util';
+import { useEffect, useState } from 'react';
+
+import useFormatReturnType from '../../hooks/useFormatReturnType';
+import usePayouts from '../NominationsPayouts/usePayouts';
+
+export default function useTotalPayoutRewards(
+  address?: string,
+  defaultValue: { value1: BN | null } = { value1: null }
+) {
+  const [value1, setValue1] = useState(defaultValue.value1);
+  const payoutsData = usePayouts();
```

Comment:
You've made a bund of queries inside the `usePayouts()` hook, and recall the hook here, so a lot of redundant calculation here, instead of that, consider passing the result into this hook and do the calculation instead of recall the `usePayouts()` hook here.